snap/snapcraft.yaml

```diff
@@ -39,11 +40,15 @@ parts:
       snapdaemon.sh: usr/bin/snapdaemon
 
 plugs:
-   etc-auto-cpufreq-conf:
+  network:
```

Comment:
Not sure why this part was added, but lines 43 to 46 should be removed. As upload of this build of this snap with these changes will otherwise fail, i.e:

```
snapcraft upload auto-cpufreq_1.9.9_amd64.snap --release beta
Issues while processing snap:g                                                                                                                                       
- unknown attribute 'allow-auto-connection' for interface 'network' (plugs)    
```

Hence this whole code block should look like just as it does on `master`:

```
plugs:
   etc-auto-cpufreq-conf:
    interface: system-files
    write:
    - /etc/auto-cpufreq.conf
```

Reply:
it was used to give network access to the container, but its not required now

Reply:
made changes according to it now